v2/storage/memory/config.go

```diff
@@ -1,10 +1,10 @@
-// Copyright (c) 2021 The Jaeger Authors.
```

Comment:
Why remove the copyright year?

Reply:
My understanding from other communities is that it isn't really required. And not having it makes it less cumbersome when the year changes.